action.yml

```diff
@@ -124,14 +124,51 @@ runs:
           FILES=$(jq -r .files[].filename __compare.json)
           rm __compare.json
           
-          # Don't allow merge-ups when there are changes in dependency files
-          DEPENDENCY_FILES="composer.json package.json yarn.lock"
+          # Don't allow merge-ups when there are changes in javascript dependency files
+          DEPENDENCY_FILES="package.json yarn.lock"
           for DEPENDENCY_FILE in $DEPENDENCY_FILES; do
             if [[ $(echo "$FILES" | grep $DEPENDENCY_FILE) != "" ]]; then
-              echo "Unable to mergeup between $FROM_BRANCH and $INTO_BRANCH - there are changes in $DEPENDENCY_FILE"
+              echo "Unable to mergeup $FROM_BRANCH into $INTO_BRANCH - there are changes in $DEPENDENCY_FILE"
               exit 1
             fi
           done
+
+          # Don't allow merge-ups where the are changes composer.json, unless the only changes are in "require-dev"
+          if [[ $(echo "$FILES" | grep composer.json) != "" ]]; then
+            RESP_CODE=$(curl -w %{http_code} -s -o __composer_from.json https://raw.githubusercontent.com/$GITHUB_REPOSITORY/$FROM_BRANCH/composer.json)
+            if [[ $RESP_CODE != 200 ]]; then
+              echo "Unable to download composer.json for branch $FROM_BRANCH"
+              exit 1
+            fi
+            RESP_CODE=$(curl -w %{http_code} -s -o __composer_into.json https://raw.githubusercontent.com/$GITHUB_REPOSITORY/$INTO_BRANCH/composer.json)
+            if [[ $RESP_CODE != 200 ]]; then
+              echo "Unable to download composer.json for branch $INTO_BRANCH"
+              exit 1
+            fi
```

Comment:
Echo our the response code as well so we know why we weren't able to download it

Reply:
Done